code/Modules/Commands/RoundCommands.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Sandbox;
+
+namespace ssl.Modules.Commands
+{
+    public class RoundCommands
+    {
+        [AdminCmd("rrestart")]
```

Comment:
rrestart looks like a typo
```suggestion
        [AdminCmd("round_restart")]
```